snap/hooks/install

```diff
@@ -42,6 +41,15 @@ sed -i \
   -e "s|<SNAP_COMMON>|${SNAP_COMMON}|g" \
   "$SNAP_DATA/nginx/nginx.conf"
 
+# Create the static directory for django
+cp -r "$SNAP/api" "$SNAP_DATA/"
+chmod -R 755 $SNAP_DATA/api
+
+# Prepare the django app
+cd $SNAP_DATA/api
+DJANGO_SECRET_KEY="testkey" python3 manage.py collectstatic --noinput
+DJANGO_SECRET_KEY="testkey" python manage.py migrate --noinput
```

Comment:
No consequence to run the migration from the install hook with a test secret key ?

Reply:
I don't think so, the secret key is mainly used to hash password and manage session tokens: https://docs.djangoproject.com/en/5.1/ref/settings/#secret-key